src/Components/SplashScreen/SplashScreen.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import { Text } from 'react-native';
 
-class SplashScreen extends Component {
+class SplashScreen extends React.Component {
```

Comment:
Probably ought to take `Component` out of the import if you are using the `React.` part.

Reply:
Fair point 👍